### PR TITLE
Playbook updates

### PR DESCRIPTION
**Pull request by glogiotatidis**
- Removes maxmind, no longer used
- Set DMS for Jenkins backups
- Updates docker cleanup script to use `docker system prune`
- Clean up and update different components

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -201,24 +243,10 @@
 
     - name: Create jenkins backup job
       cron: name="Backup Jenkins" minute=0 hour=4
-            user="root" job="s3cmd -c /home/{{ remote_user }}/.s3cfg put -r /var/lib/jenkins/backups/*.zip s3://{{ aws_backup_bucket }} --encrypt 2> /dev/null"
+            user="root"
+            job="(s3cmd -c /home/{{ remote_user }}/.s3cfg put -r /var/lib/jenkins/backups/*.zip s3://{{ aws_backup_bucket }} --encrypt 2> /dev/null) && curl -s  {{ jenkins_backup_dms }}"
```
Comment (bookshelfdave) [inline site.yml:247]: are AWS creds stored on disk? We've been using the aws cli w/ the [AWS metadata service](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#instance-metadata-security-credentials) so creds aren't stored locally. Probably fine as-is though.

**Comment by bookshelfdave** [comment]
looks good @glogiotatidis 
👍

**Comment by glogiotatidis** [comment]
thanks @metadave

File: site.yml (hunk 1 above)
Comment (glogiotatidis) [reply site.yml:247]: yes those are stored on disk

**Pull request merged**